src/drive/web/modules/views/Public/index.spec.jsx

```diff
@@ -68,5 +75,11 @@ describe('Recent View', () => {
     expect(
       document.querySelectorAll("[data-test-id='fil-actionmenu-inner']").length
     ).toEqual(1)
+
+    // navigates  to the history view
+    const historyItem = getByText('History')
+    fireEvent.click(historyItem)
+
+    expect(findByText('Current version')).toBeTruthy()
```

Comment:
findBy methods need to be awaited. Here you are checking that a Promise is truthy.

You can 

```
await findByText('Current version')

// or the more explicit
await expect(findByText('Current version')).toResolve()
```